docker-compose-local.yml

```diff
@@ -0,0 +1,112 @@
+version: '3.1'
```

Comment:
Only comment is that this could all go in `docker-compose.yml` and you can just start the services you require by naming them in `docker-compose up`. That's all we do, usual for me is `docker-compose up redis-bapv` as that's all I need 🙂